pkg/store/badger_cache.go

```diff
@@ -0,0 +1,91 @@
+package store
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/finitum/AAAAA/pkg/aur"
+	"github.com/pkg/errors"
+	"time"
+)
+
+type BadgerCache struct {
+	db *badger.DB
+}
+
+const userPrefix = "user_"
+const cacheTTL = 30 * time.Minute
+
+func OpenBadgerCache(path string) (*BadgerCache, error) {
+	db, err := badger.Open(badger.DefaultOptions(path + ".cache"))
+	if err != nil {
+		return nil, errors.Wrap(err, "opening badger store")
+	}
+
+	// TODO: Schedule garbage collection
```

Comment:
I was under the impression that badger did this itself, is that incorrect?

Reply:
that's incorrect. You have to call the GC function or close the DB (that also collects garbage)

Reply:
https://dgraph.io/docs/badger/get-started/#garbage-collection

Reply:
We should do that then